### ISSUE TITLE
BrowserStack IE error SCRIPT1002: Syntax Error

### ISSUE THREAD

**Issue by mdonjones** [bugsnag/bugsnag-js#552]
Hey all, 

We've been using:
```
    "@bugsnag/js": "^6.0.0",
    "@bugsnag/plugin-angular": "^6.0.0",
```
for about two months now - and recently we ran into the `SCRIPT1002: Syntax Error` while testing IE11 in browserstack. Our build that had been in production was running fine with no changes, but our local instances were all running into this issue. We tracked it down to the recently released `6.3.0` version of this bugsnag library. Once we removed the `^` from the version and explicitly used version `6.0.0` it worked fine, as does `6.2.0`.

This is all I really have to provide for screenshots, the error and where the error pointed to:
![image](https://user-images.githubusercontent.com/1068742/59356311-f7e3e400-8cfe-11e9-8276-7ab5b3c1fa89.png)
![image](https://user-images.githubusercontent.com/1068742/59356365-13e78580-8cff-11e9-9186-7164989168c8.png)

This could potentially be related to #550 ?

**Comment by bengourley** [comment]
It looks like it's complaining about the native classes – we switched to using ES6 classes in [v6.3.0](https://github.com/bugsnag/bugsnag-js/releases/tag/v6.3.0) (in the Angular plugin only).

What is `target` set to in your `tsconfig.json`? If you want to support IE11 I think that'll need to be set to `"es5"` – can you confirm that it is?

**Comment by mdonjones** [comment]
@bengourley I can confirm it is currently set to `"target": "es5"`

**Comment by bengourley** [comment]
Ok – that suggests the TypeScript compiler is not transforming this module at all, perhaps due to it already being JS.

The switch fixed a different bug for users who were targetting ES6. We'll investigate and see what we can do to solve both problems.

**Comment by bengourley** [comment]
Please try the prerelease as outlined in https://github.com/bugsnag/bugsnag-js/pull/565#issuecomment-505429174 and confirm whether this solves the issue. Thanks!

**Comment by abigail-bugsnag** [comment]
This is fixed as of v6.3.2 (https://github.com/bugsnag/bugsnag-js/releases/tag/v6.3.2)
